### PR TITLE
Fix file name and make parameters more understandable

### PR DESCRIPTION
**Pull request by OPpuolitaival**

### CODE REVIEW

File: ostree/README.md [hunk 1]
```diff
@@ -28,19 +28,19 @@ Notes:
...
 ```
-> ./ostree-delta.py <--repo repo> <--output output-dir> [--update_repo repo] [--to_sha sha] [--from_sha sha] [--commit message] [--generate_bin]
+> ./ostree-delta.py --repo repo --output output-dir [--update_repo repo] [--to_sha sha] [--from_sha sha] [--commit message] [--generate_bin]
 ```
 
 If using the Docker container use:
 
 ```
-> ./ostree-delta-docker.sh <--repo=repo> <--output=output-dir> [--update_repo repo] -- [--to_sha sha] [--from_sha sha]  [--commit message] [--generate_bin]
+> ./ostree-delta-docker.sh --repo=repo --output=output-dir [--update_repo repo] -- [--to_sha sha] [--from_sha sha]  [--commit message] [--generate_bin]
```
Comment (petedyerarm) [inline ostree/README.md:37]:
```suggestion
> ./ostree-delta-docker.sh --repo repo --output output-dir [--update_repo repo] -- [--to_sha sha] [--from_sha sha]  [--commit message] [--generate_bin]
```

**Review by petedyerarm — CHANGES_REQUESTED** [review]

File: ostree/README.md (hunk 1 above)
Comment (cmonr) [reply ostree/README.md:37]: :+1:

**Review by cmonr — APPROVED** [review]

**Comment by cmonr** [comment]
Bringing in. Tweaks look good.

**Review by cmonr — APPROVED** [review]

**Comment by cmonr** [comment]
Huh. Actually, something looks weird with the view. @petedyerarm, are you able to do anything on your end?

PR says it's waiting on your change request, but imo it was taken in with the new commit.

**Review by petedyerarm — APPROVED** [review]

**Pull request merged**